releases/release-5.3.0.md

```diff
@@ -0,0 +1,392 @@
+
+### Security
+
+- **Support creating users with the least privileges on TiDB Dashboard**
+
+The account system of TiDB Dashboard is consistent with that of TiDB SQL. Users accessing TiDB Dashboard are authenticated and authorized based on TiDB SQL users' privileges. Therefore, TiDB Dashboard requires limited privileges, or merely the read-only privilege. You can configure users to access TiDB Dashboard based on the principle of least privilege, thus avoiding access of high-privileged users.
+
+It is recommended that you create a least-privileged SQL user to access and sign in with TiDB Dashboard. This avoids access of high-privileged users and improves security.
+
+[User document](/dashboard/dashboard-user.md)
+
+### Performance
+
+- **Optimize the timestamp processing flow of PD**
+
+    TiDB optimizes its timestamp processing flow and reduces the timestamp processing load of PD by enabling PD Follower Proxy and modifying the batch waiting time required when the PD client requests TSO in batches. This helps improve the overall scalability of the system.
+
+    - Support enabling or disabling PD Follower Proxy through the system variable [`tidb_enable_tso_follower_proxy`](/system-variables.md#tidb_enable_tso_follower_proxy-new-in-v53). Suppose that the TSO requests load of PD is too high. In this case, enabling PD follower proxy can batch forward the TSO requests collected during the request cycle on followers to the leader nodes. This solution can effectively reduce the number of direct interactions between clients and leaders, reduce the pressure of the load on leaders, and improve the overall performance of TiDB. 
+
+    > **Note:**
+    >
+    > When the number of clients is small and the PD leader CPU is not full, it is NOT recommended to enable PD Follower Proxy.
+
+    - Support using the system variable [`tidb_tso_client_batch_max_wait_time`](/system-variables.md#tidb_tso_client_batch_max_wait_time-new-in-v53) to set the maximum waiting time needed for the PD client to batch request TSO. The unit of this time is milliseconds. In case that PD has a high TSO requests load, you can reduce the load and improve the throughput by increasing the waiting time to get a larger batch size.
+
+    > **Note:**
+    >
+    > When the TSO request load is not high, it is NOT recommended to modify this variable value.
+
+    [User document](/system-variables.md#tidb_tso_client_batch_max_wait_time-new-in-v53), [#3149](https://github.com/tikv/pd/issues/3149)
+
+### Stability
+
+- **Support Online Unsafe Recovery after some stores are permanently damaged (experimental feature)**
+
+    Support the `unsafe remove-failed-stores` command that allows data unsafe recovery. Suppose that multiple replicas of data encounter problems like permanent damage (such as disk damage), and this issue causes the data in an application to be not readable or writable. In this case, Online Unsafe Recovery using PD can recover the data to be readable or writable again.
```

Comment:
Might be good to list the full command here. Is this `pd-ctl unsafe remove-failed-stores`? How does this related to `tikv-ctl 
unsafe-recover remove-fail-stores`?

As I understand this this improves this by:

- being online
- uses PD to send commands to the stores as opposed to have to run the commands on the stores themselves

So there already was `Unsafe Recovery`, but now we have `Online Unsafe Recovery`. This wasn't clear for the release notes.

Reply:
Thanks, Daniël! @Connor1996 could you please help check how should we update the command?

Reply:
Support the `pd-ctl unsafe remove-failed-stores` command that allows data unsafe recovery online which is a replacement of `tikv-ctl unsafe-recover remove-fail-stores`.